speclang_modern.tex

```diff
@@ -1280,6 +1303,7 @@ \subsection{Memory locations and sets of terms}
   satisfying predicate \lstinline|P|
   (binders \lstinline|b| are bound in both \lstinline|s| and \lstinline|P|).
 \item \lstinline|x \in s| holds if and only if \lstinline|x| is an element of \lstinline|s|.
+  The operator has the same precedence as relational predicates (e.g., \lstinline|<|).
```

Comment:
`\in` is not implemented with that precedence into the parser Frama-C 19.0 (Potassium). However, an issue has been identified about the implemented precedence rules. So, the proposed precedence rule will be reviewed later in combination with the corresponding Frama-C Merge Request (the proposition can be seen as a feature request for that MR).

Reply:
As mentioned in the Frama-C MR, this precedence is fine